### PR TITLE
Deploy qci-appci

### PR DESCRIPTION
**Pull request by hongkailiu**
https://issues.redhat.com/browse/DPTP-3749

### CODE REVIEW

**Comment by hongkailiu** [comment]
/hold

require https://github.com/openshift/ci-tools/pull/3773

File: clusters/app.ci/assets/admin_qci-appci.yaml [hunk 1]
```diff
@@ -35,3 +66,54 @@ spec:
             port:
               number: 8400
         pathType: ImplementationSpecific
+---
+apiVersion: apps/v1
+kind: Deployment
```
Comment (smg247) [inline clusters/app.ci/assets/admin_qci-appci.yaml:71]: The deployment should probably go in its own file

**Pull request merged**